Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/interface/.DS_Store and wxnew/interface/.DS_Store differ
Binary files wxorig/interface/wx/.DS_Store and wxnew/interface/wx/.DS_Store differ
diff -uNr wxorig/interface/wx/event.h wxnew/interface/wx/event.h
--- wxorig/interface/wx/event.h	2011-02-05 20:30:36.000000000 -0500
+++ wxnew/interface/wx/event.h	2011-03-18 15:42:58.000000000 -0400
@@ -2890,11 +2890,11 @@
 /**
     @class wxMouseCaptureLostEvent
 
-    An mouse capture lost event is sent to a window that obtained mouse capture,
-    which was subsequently loss due to "external" event, for example when a dialog
-    box is shown or if another application captures the mouse.
+    A mouse capture lost event is sent to a window that had obtained mouse capture,
+    which was subsequently lost due to an "external" event (for example, when a dialog
+    box is shown or if another application captures the mouse).
 
-    If this happens, this event is sent to all windows that are on capture stack
+    If this happens, this event is sent to all windows that are on the capture stack
     (i.e. called CaptureMouse, but didn't call ReleaseMouse yet). The event is
     not sent if the capture changes because of a call to CaptureMouse or
     ReleaseMouse.
